R/exportToJson.R

```diff
@@ -20,22 +20,24 @@
 # @author Chris Knoll
 # @author Frank DeFalco
 
+
+# Run this definition of allReports when adding a new report
 # allReports <- c("CONDITION",
-#                "CONDITION_ERA",  
-#                "DASHBOARD", 
+#                "CONDITION_ERA",
+#                "DASHBOARD",
 #                "DATA_DENSITY",
 #                "DEATH",
 #                "DRUG",
 #                "DRUG_ERA",
 #                "HEEL",
 #                "OBSERVATION",
 #                "OBSERVATION_PERIOD",
-#                "PERSON", 
+#                "PERSON",
 #                "PROCEDURE",
 #                "VISIT",
 #                "MEASUREMENT",
 #                "META")
-# save(allReports,file="allReports.rda")
+# save(allReports,file="data/allReports.rda")

```

Comment:
@chrisknoll in the checked in version I forked, the allReports.rds file was in "/data", so I just assumed that's where it was supposed to go. Then I changed the path to allReports in the commented section above.